indigo_app/views/places.py

```diff
@@ -553,6 +561,19 @@ def form_valid(self, form):
         placesettings = self.object
         placesettings.updated_by_user = self.request.user
 
+        # save changes to italics_terms 1) as a list 2) to the country
+        if 'italics_terms' in form.changed_data:
```

Comment:
It feels like doing this in the form is the better place to do it. Perhaps https://docs.djangoproject.com/en/2.2/ref/contrib/postgres/forms/#simplearrayfield would simplify this?

Reply:
I've pushed a fix to do this, reduces our logic a bunch.